packages/theming/atlas/scripts/build.js

```diff
@@ -63,12 +64,18 @@ async function buildAndCopyAtlas(watchMode, destination) {
                 {
                     name: "native-design-properties-and-manifest",
                     command: `copy-and-watch ${watchArg} "src/themesource/atlas_core/native/**/*.json" "${destination}/themesource/atlas_core/native"`
+                },
+                {
+                    name: "atlas-2-to-3-migration-tool",
+                    command: `copy-and-watch ${watchArg} "src/themesource/atlas_core/migration/*" "${destination}/themesource/atlas_core/migration"`
                 }
             ],
             {
                 killOthers: ["failure"]
             }
         );
+        // This needs to be done after the concurrently batch above, since we're dependent on the resulting `theme` folder.
+        await zip(`${destination}/theme`, `${destination}/themesource/atlas_core/migration/theme.zip`);
```

Comment:
right now it's done on `npm run start` `npm run build` and `npm run release`. Is that necessary?

Reply:
That's what we want.

Reply:
I originally thought just release, but it makes more sense to go with what [Isa said](https://github.com/mendix/widgets-resources/pull/522/files#r595154739) is a better workflow.

release -> dist
start | build -> test project